templates/zerver/help/view-an-image-at-full-size.md

```diff
@@ -11,6 +11,6 @@ Your screen will look like this:
 ![full-size](/static/images/help/full-size.png)
 
 From here, you can also open the image in a new tab or download
-it. These options can be seen in the top right of your screen.
+it. These options can be seen on the top right of your screen.
 
 Click the **X** in the top right to return to your Zulip chat.
```

Comment:
I know this isn't in the scope of your PR, but you can change `Click the **X** in the top right to return to your Zulip chat.` to use the icon format mentioned in the styling guides.